.gitignore

```diff
@@ -76,8 +76,8 @@ orc8r/cloud/helm/**/.secrets/
 # Certs for accessing the cloud
 .certs
 
-# CMake debug files
-lte/gateway/c/session_manager/cmake-build-debug/
+# CMake debug files; copied into source tree by CLion when using remote CMake
+lte/gateway/c/**/cmake-build-debug
```

Comment:
note this directory depends on the build directory defined on clion so I never added it.

See below how many different flavors I currently have on my mac

```
#      untracked:  [4] c/core/oai/cmake-build-debug/
#      untracked:  [5] c/oai/cmake-build-debug/
#      untracked:  [6] c/session_manager/cmake-build-debug-remote-host-2222/
#      untracked:  [7] c/session_manager/cmake-build-debug-remote-host_default/
```

Would something like this work?
```
lte/gateway/c/**/cmake-build-**
```


![image](https://user-images.githubusercontent.com/16157139/125718332-b35f91db-8355-4ec3-9eac-d20fa91550e8.png)